tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -462,7 +462,12 @@ task MergeMetrics {
 
     for col in range(0, len(rows[0])):
       key = rows[0][col].lower()
-      print(f"{key}\t{rows[1][col]}")
+      value = rows[1][col]
+      if value == "?":
+        value = "NaN"
+      if key in ["median_insert_size", "median_absolute_deviation", "median_read_length", "hq_median_mismatches"]:
+        value = str(int(float(value)))
```

Comment:
don't need to cast to a `str`, the f-string will take care of this for you.

Reply:
yeah, I went back and forth on this - I left it with the str() conversion just to return `value` to the same type as it started as. if you have strong feelings about this, happy to change it.